rs/universal_canister/lib/src/lib.rs

```diff
@@ -7,17 +7,37 @@
 
 pub mod management;
 
-use hex_literal::hex;
 use ic_types::Cycles;
 use universal_canister::Ops;
-/// The binary of the universal canister as compiled from
-/// `rs/universal_canister/impl`.
-///
-/// For steps on how to produce it, please see the README in
-/// `rs/universal_canister`.
-pub const UNIVERSAL_CANISTER_WASM: &[u8] = include_bytes!("universal-canister.wasm.gz");
+
+/// The binary of the universal canister as compiled from `rs/universal_canister/impl`.
+pub const UNIVERSAL_CANISTER_WASM: &[u8] = include_bytes!(env!("UNIVERSAL_CANISTER_WASM_PATH"));
 pub const UNIVERSAL_CANISTER_WASM_SHA256: [u8; 32] =
```

Comment:
I'd turn this into a function since it isn't used a lot in the code base and thus refactoring seems feasible.